doc/overviews/dns.md

```diff
@@ -29,6 +29,8 @@ A DNSPolicy acts against a target Gateway by processing its listeners for hostna
 In order for it to do this, it must know about the dns provider.
 This is done through the creation of dns provider secrets containing the credentials and configuration for the dns provider account.
 
+The policy can use a specific provider secret by referencing it in the `ProviderRefs`. Alternatively, if no reference is provided, the secret with `kuadrant.io/default-provider` label will be chosen as a default option.
```

Comment:
I think it would be useful to call out here there can only be one default provider. Also I think we should add something to 
https://github.com/Kuadrant/dns-operator/blob/main/docs/provider.md